snowflake/1.1/modules/ROOT/pages/snowflake-connector-studio.adoc

```diff
@@ -43,11 +43,29 @@ Adding a connector to a Mule project in Studio does not make that connector avai
 A source initiates a flow when a specified condition is met.
 You can configure one of these sources to use with Snowflake Connector:
 
+* Snowflake > On Table Row +
+Initiates a flow by selecting a table at a regular interval and generating one message per obtained row
 * HTTP Listener +
 Initiates a flow each time it receives a request on the configured host and port
 * Scheduler +
 Initiates a flow when a time-based condition is met
 
+For example, to configure an *On Table Row* source, follow these steps:
+
+. In *Mule Palette*, select *Snowflake > On Table Row*.
+. Drag *On Table Row* to the Studio canvas.
+. On the *On Table Row* configuration screen, optionally change the value of *Display Name*.
+. Click *+* next to *Connector configuration* to configure a global element that can be used by all instances of the *On Table Row* source in the app.
+. On the *Snowflake Config* window, for *Required Libraries*, select the `snowflake-jdbc` Maven dependency.
```

Comment:
On the *Snowflake Config* window, for *Required Libraries*, add the required Snowflake JDBC driver clicking on `Configure > Add recommended libraries` or `Configure > Add Maven Dependency` searching by artifact id as `snowflake-jdbc`.